code/frameworks/nextjs-server/template/app/storybook-preview/components/previewHtml.ts

```diff
@@ -0,0 +1,418 @@
+export const previewHtml = `
```

Comment:
This code is now in 3 places, we should figure out how to share it.

NOTE this one is slightly different as I removed the `#storybook-root` element.

Reply:
why did you remove the `#storybook-root` element in this one?

Reply:
Because it is rendered by the route in the /app dir case. That's different to the usual situation where the preview picks off the static dom element and passes it to the renderer